src/css/job-posts.css

```
@@ -0,0 +1,317 @@
+.job-posts {
+  overflow: hidden;
+  background: url(/src/backgound-svg/2-background.svg) var(--COLOR_DARK_BLUE)
+    center;
```

```suggestion
  background: url(/src/backgound-svg/2-background.svg) var(--COLOR_DARK_BLUE)
    center;
```
No need in this style